tests/FakeItEasy.Specs/ConfigurationSpecs.cs

```diff
@@ -184,20 +184,255 @@ public interface IFoo
                 .x(() => exception.Should().BeAnExceptionOfType<InvalidOperationException>());
         }
 
+        [Scenario]
+        public static void CallToObjectOnNonFake(
+            BaseClass notAFake,
+            Exception exception)
+        {
+            "Given an object that is not a fake"
+                .x(() => notAFake = new BaseClass());
+
+            "When I start to configure the object"
+                .x(() => exception = Record.Exception(() => A.CallTo(notAFake)));
+
+            "Then it throws an argument exception"
+               .x(() => exception.Should().BeAnExceptionOfType<ArgumentException>()
+                   .And.Message.Should().Contain("The specified object is not recognized as a fake object."));

```

Comment:
Not a bad idea. Many of the the exception messages are already in resources. Although I'm funny... I don't mind them being repeated as literals so much in the tests (I wouldn't fight if you wanted to use the resource in the test as well though). It's when I see them repeated in the production code that I'm less comfortable.